stable/traefik/README.md

```diff
@@ -107,6 +107,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.email`                    | Email address to be used in certificates obtained from Let's Encrypt | `admin@example.com`                       |
 | `acme.staging`                  | Whether to get certs from Let's Encrypt's staging environment        | `true`                                    |
 | `acme.logging`                  | display debug log messages from the acme client library              | `false`                                   |
+| `acme.httpChallenge`                  | enables HTTP-01 challenge to generate the certificates              | `false`                                   |
```

Comment:
There is a PR open to add a DNS challenge. See #1299.

Taking this into consideration and assuming that _other_ types of challenges could be supported in the future as well, we shouldn't use a boolean for opting into one particular sort of challenge. Instead, we should use a discriminator that identifies _which_ of many possible challenge types should be used.

In [this comment](https://github.com/kubernetes/charts/pull/1299#issuecomment-360379019) I have proposed using `acme.challengeType`

Reply:
I can do this change no problem.